.circleci/config.yml

```diff
@@ -1,209 +1,167 @@
-    - store_test_results:
-        path: test_results
-
-  build-image:
-    executor: docker-publisher
-    steps:
-    - checkout
-    - setup_remote_docker:
-        # Setting docker version per https://support.circleci.com/hc/en-us/articles/360050934711-Docker-build-fails-with-EPERM-operation-not-permitted-copyfile-when-using-node-14-9-0-or-later-
-        version: 19.03.13
-    - run:
-        name: Build Docker image
-        command: |
-          docker build -t $IMAGE_NAME:latest -f docker/app/Dockerfile .
-    - run:
-        name: Archive Docker image
-        command: |
-          docker save -o app_image.tar $IMAGE_NAME
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ./app_image.tar
-  publish-latest:
-    executor: docker-publisher
-    steps:
-    - attach_workspace:
-        at: /tmp/workspace
-    - setup_remote_docker:
-        # Setting docker version per https://support.circleci.com/hc/en-us/articles/360050934711-Docker-build-fails-with-EPERM-operation-not-permitted-copyfile-when-using-node-14-9-0-or-later-
-        version: 19.03.13
-    - run:
-        name: Load archived Docker image
-        command: |
-          docker load -i /tmp/workspace/app_image.tar
-    - run:
-        name: Publish Docker Image to Docker Hub
-        command: |
-          echo "$DOCKER_PASS" | docker login -u "$DOCKER_USER" --password-stdin
-          docker push $IMAGE_NAME:latest
-  publish-tag:
-    executor: docker-publisher
-    steps:
-    - attach_workspace:
-        at: /tmp/workspace
-    - setup_remote_docker:
-        # Setting docker version per https://support.circleci.com/hc/en-us/articles/360050934711-Docker-build-fails-with-EPERM-operation-not-permitted-copyfile-when-using-node-14-9-0-or-later-
-        version: 19.03.13
-    - run:
-        name: Load archived Docker image
-        command: |
-          docker load -i /tmp/workspace/app_image.tar
-    - run:
-        name: Publish Docker Image to Docker Hub
-        command: |
-          echo "$DOCKER_PASS" | docker login -u "$DOCKER_USER" --password-stdin
-          docker tag $IMAGE_NAME:latest $IMAGE_NAME:$CIRCLE_TAG
-          docker push $IMAGE_NAME:$CIRCLE_TAG
-workflows:
-  version: 2
-
-  test:
+orbs:
+  ruby-rails:
+    orbs:
+      docker: circleci/docker@2.0.1
+      node: circleci/node@3.0
+      ruby: circleci/ruby@1.2.0
+    commands:
+      validate-api:
+        description: Validate openapi specification
+        steps:
+          - run:
+              name: Validate API specification
+              command: |
+                sudo npm install -g openapi-enforcer-cli
+                result=$(openapi-enforcer validate openapi.yml)
+                [[ $result =~ "Document is valid" ]] && {
+                echo "Validation good"
+                exit 0
+                } || {
+                echo $result
+                exit 1
+                }
     jobs:
-    - test
-
+      build:
+        parameters:
+          tag:
```

Comment:
Can this be `ruby_tag`? This is what argo already uses: https://github.com/sul-dlss/argo/blob/main/.circleci/config.yml#L70